frontend/src/screens/researcher/studies/overview/finalize-study.tsx

```diff
@@ -82,21 +83,103 @@ const QualtricsConfirmationContainer: FC<{
 
     if (study.stages?.length === 1) {
         return (
-            <QualtricsConfirmation gap align='center'>
-                <ResearcherCheckbox type='checkbox' name='userHasCheckedQualtrics.0' data-testid='confirm-qualtrics' id='confirm-qualtrics'/>
-                <label htmlFor='confirm-qualtrics'>Yes, I have set up my study in Qualtrics</label>
-            </QualtricsConfirmation>
+            <div>
+                <QualtricsConfirmation gap align='center'>
+                    <ResearcherCheckbox type='checkbox' name='userHasCheckedQualtrics.0' data-testid='confirm-qualtrics' id='confirm-qualtrics'/>
+                    <label htmlFor='confirm-qualtrics'>Yes, I have set up my study in Qualtrics</label>
+                </QualtricsConfirmation>
+
+                <div style={{ height: '50px' }}></div>
+            </div>
         )
     }
 
     return (
         <div>
             {study.stages?.map((stage, index) => (
-                <QualtricsConfirmation gap align='center' key={stage.order} className='mt-1'>
-                    <ResearcherCheckbox type='checkbox' name={`userHasCheckedQualtrics.${index}`} data-testid={`confirm-qualtrics-${index}`} id={`confirm-qualtrics-${index}`} />
-                    <label htmlFor={`confirm-qualtrics-${index}`}>Yes, I have set up Session {index + 1} in Qualtrics</label>
-                </QualtricsConfirmation>
+                <div key={stage.order} className='mt-1'>
+                    <QualtricsConfirmation gap align='center'>
+                        <ResearcherCheckbox type='checkbox' name={`userHasCheckedQualtrics.${index}`} data-testid={`confirm-qualtrics-${index}`} id={`confirm-qualtrics-${index}`} />
+                        <label htmlFor={`confirm-qualtrics-${index}`}>Yes, I have set up Session {index + 1} in Qualtrics</label>
+                    </QualtricsConfirmation>
+                    {index === (study.stages?.length ?? -1) - 1 && (
+                        <div style={{ height: '50px' }}></div>
+                    )}
+                </div>
             ))}
         </div>
     )
 }
+
+export const ExitWithConfirmation: FC<{navTo: string}> = ({ navTo }) => {
+    const [showWarning, setShowWarning] = useState(false);
+    const nav = useNavigate();
+
+    const handleExit = () => {
+        if (window.confirm("You're about to leave this study creation process.")) {
+            // Perform the save operation here
+            // Example: saveStudy(getValues() as Study);
+        } else {
+            nav(navTo); // Use nav to navigate to the specified page
+        }
+    };
+
+    return (
+        <div>
+            <h6
+                style={{
+                    textDecoration: 'underline',
+                    textUnderlineOffset: '.5rem',
+                    cursor: 'pointer',
+                }}
+                onClick={() => {
+                    if (showWarning) {
+                        setShowWarning(false);
```

Comment:
Currently the warning never gets triggered by the form, this is an inherent problem with the form context only being available "inside" the form. since this button is outside, there doesn't seem to be an obvious way to get the forms state (`dirty` in this case)

I wonder if we can talk to iris/viki and change the approach. If we could get away with a `Window.confirm()` and utilize the default browser rather than rolling our own. That would solve this problem in a couple different areas

@vikizhu5504 @iccgoncalves